views/apartment.erb

```diff
@@ -0,0 +1,5 @@
+<%="Address: #{@apartment.address}<br>
```

Comment:
Clown hats should only be one line each! If you want to have stuff on multiple lines, use multiple clown hats.